src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/GCStaticEETypeNode.cs

```diff
@@ -87,23 +68,13 @@ int ISymbolNode.Offset
         {
             get
             {
-                if (NumSeries > 0)
-                {
-                    return _targetPointerSize * ((NumSeries * 2) + 1);
-                }
-                else
-                {
-                    return 0;
-                }
+                return ((_gcMap.NumSeries * 2) + 1) * _target.PointerSize;
```

Comment:
It would be nice to do this in GCDescEncoder - since all other encoding details are encapsulated in it.